extensions/wikia/Search/classes/IndexService/CrossWikiCore.php

```diff
@@ -154,6 +161,23 @@ protected function getVisualizationInfo() {
 		return $response;
 	}
 
+	protected function getCommunityData() {
+		$useUnifiedSearch = ( new UnifiedSearchService() )->useUnifiedSearch( true );
+		if ( $useUnifiedSearch ) {
+			$communityData = ( new CommunityDataService( $this->wikiId ) )->getCommunityData();
+
+			if ( !empty( $communityData['image_id'] ) ) {
+				$url = CuratedContentHelper::getImageUrl( $communityData['image_id'], 500 );
```

Comment:
`500` - this is image size - how big this image should be?

Reply:
LGTM